.travis.yml

```diff
@@ -7,13 +7,11 @@ env:
   global:
   - DOCKER_REPO=nhsuk/${REPLACEME}
   - DOCKER_USERNAME=nhsukautomata
-  - RANCHER_ACCESS_KEY=655696CC96424E6D4DB0
-  - RANCHER_ENVIRONMENT=c2s-dev
   - RANCHER_TEMPLATE_NAME=${REPLACEME}
-  - RANCHER_URL=https://rancher.nhschoices.net
+  - RANCHER_URL=${URL-https://rancher.nhschoices.net}
```

Comment:
The file contains examples, as explained in the README. The change is simply to make people think about changing it rather than blindly accepting the default values.